### PR TITLE
feat(fe2): show reqId in error pages

### PR DESCRIPTION
**Pull request by fabis94**
## Checklist:

- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

### CODE REVIEW

File: packages/frontend-2/components/project/model-page/versions/Card.vue [hunk 1]
```diff
@@ -40,7 +40,11 @@
               <PreviewImage :preview-url="version.previewUrl" />
             </NuxtLink>
             <div
-              v-if="!isPendingVersionFragment(version) && version.automationsStatus"
+              v-if="
```
Comment (fabis94) [inline packages/frontend-2/components/project/model-page/versions/Card.vue:43]: @Mikehrn I already hid the donut in a couple of places, cause it was ruining a screen recording im doing for marketing

File: packages/frontend-2/components/error/page/GenericErrorBlock.vue [hunk 2]
```diff
@@ -3,18 +3,24 @@
...
     <!-- <ErrorPageProjectInviteBanner /> -->
     <h1 class="h1 font-medium">Error {{ error.statusCode || 500 }}</h1>
-    <h2 class="h3 text-foreground-2 mx-4 break-words max-w-full">
-      {{ error.message }}
-    </h2>
+    <div class="flex flex-col items-center space-y-1">
+      <h2 class="h2 text-foreground-2 mx-4 break-words max-w-full">
+        {{ error.message }}
+      </h2>
+      <div class="text-foreground-2 text-xs">Reference #{{ reqId }}</div>
```
Comment (Mikehrn) [inline packages/frontend-2/components/error/page/GenericErrorBlock.vue:10]: Could you use the new text class here? `text-body-xs` or `text-body-2xs` probably :D

**Review by Mikehrn — CHANGES_REQUESTED** [review]

**Review by gjedlicska — APPROVED** [review]

File: packages/frontend-2/components/project/model-page/versions/Card.vue (hunk 1 above)
Comment (Mikehrn) [reply packages/frontend-2/components/project/model-page/versions/Card.vue:43]: Nice, thanks!

File: packages/frontend-2/components/error/page/GenericErrorBlock.vue (hunk 2 above)
Comment (fabis94) [reply packages/frontend-2/components/error/page/GenericErrorBlock.vue:10]: @Mikehrn I wanted it to be even smaller than text-body-2xs, really want it to be as tiny as possible 🤔

Comment (Mikehrn) [reply packages/frontend-2/components/error/page/GenericErrorBlock.vue:10]: @fabis94 There is also -3xs, if that's still too big lets go with this then :)

**Review by Mikehrn — DISMISSED** [review]

**Review by Mikehrn — APPROVED** [review]

**Pull request merged**